dynamic-layers/qt5-layer/recipes-browser/wpewebkit/wpewebkit_%.bbappend

```diff
@@ -1,3 +0,0 @@
-# Enable QT bindings for WPE when meta-qt5 is in layer mix
-#
-PACKAGECONFIG_append = " qtwpe"
```

Comment:
This implies a change in the behaviour of this meta-layer. This config was originally added by @philn. Let's wait for his input.

Reply:
And I was the one asking this change :)